src/core/public/chrome/ui/header/nav_drawer.tsx

```diff
@@ -50,11 +46,7 @@ function navDrawerRenderer(
         defaultMessage: 'Primary',
       })}
     >
-      {RecentLinks({
-        recentlyAccessedItems,
-        navLinks: chromeNavLinks,
-        basePath,
-      })}
+      {RecentLinks({ recentNavLinks })}
```

Comment:
Can't be written as a JSX component because `NavDrawer` expects v specific children (and the Fragment which gets "rendered" then blows everything up.

Not going to bother moving this file around because it'll be deleted when the legacy nav is removed anyways.